### PR TITLE
Update the default value of `db.tx_log.rotation.retention_policy`

### PR DESCRIPTION
**Pull request by NataliaIvakina**
In Neo4j 5.13, the def.value of the config `db.tx_log.rotation.retention_policy` was changed from `2 days` to `2 days 2G`. To reflect those changes, we have to update two pages in the Operations manual.

### CODE REVIEW

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 1]
```diff
@@ -4654,6 +4654,7 @@ m|+++true+++
 === `db.tx_log.rotation.retention_policy`
 
 label:dynamic[Dynamic]
+label:changed[The default value was changed in 5.13]
```
Comment (renetapopova) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:4657]:
```suggestion
```

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 2]
```diff
@@ -4654,6 +4654,7 @@ m|+++true+++
 === `db.tx_log.rotation.retention_policy`
 
 label:dynamic[Dynamic]
```
Comment (renetapopova) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:4656]:
```suggestion
label:dynamic[Dynamic] label:changed[Default value changed in in 5.13]
```

**Review by renetapopova — APPROVED** [review]
I have a small suggestion just to be consistent with the labels of the other configs. Otherwise, looks good.

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 3]
```diff
@@ -4653,7 +4653,7 @@ m|+++true+++
 [[config_db.tx_log.rotation.retention_policy]]
 === `db.tx_log.rotation.retention_policy`
 
-label:dynamic[Dynamic]
+label:dynamic[Dynamic] label:changed[Default value changed in in 5.13]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:4656]:
```suggestion
label:dynamic[Dynamic] label:changed[Default value changed in 5.13]
```

**Pull request merged**